src/hooks/eligibility.tsx

```diff
@@ -262,7 +262,9 @@ function eligibilityYearBuilt(criteriaData: CriteriaData): CriteriaEligibility {
       userValue = (
         <>
           {`There is no recorded certificate of occupancy for your building since
-          ${cutoffYear}, ${determination === "unknown" ? "but" : "and"} other data
+          ${cutoffYear}, ${
+            determination === "unknown" ? "but" : "and"
+          } other data
           indicates it was built in {yearbuilt}.`}
```

Comment:
this might have to be `${yearbuilt}`

<img width="332" alt="Screenshot 2024-11-08 at 12 55 48 PM" src="https://github.com/user-attachments/assets/99786431-fff2-42ce-982e-82499c279381">